src/app/SOCestimator/BatteryCharacteristics.h

```diff
@@ -0,0 +1,23 @@
+#ifndef BATTERY_CHARACTERISTICS_H
+#define BATTERY_CHARACTERISTICS_H
+
+#define NUM_TEMP_RANGES 5
+
+// columns: maximum SOC
+// rows: average voltage of series cell
+// example: row 0 col 0 means a voltage of 4.0V indicates the battery is at no more than 75% SOC when its <= 5 deg C
+// each index is 0-10 deg C
+static int temp_lut_V[NUM_TEMP_RANGES][3] = 
+{//  75%  50%  25%
+    {4.0, 3.8, 3.7}, //       t <= 5C
+    {3.8, 3.7, 3.6}, //  5C < t <= 15C
+    {3.8, 3.6, 3.5}, // 15C < t <= 25C
+    {3.8, 3.5, 3.4}, // 25C < t <= 35C
+    {3.5, 3.4, 3.3}  //       t > 35C
+}
+
+#define SOC_75_LIMIT_FOR_TEMP_INDEX(temp_index) temp_lut_V[temp_index][0]
```

Comment:
Any reason for choosing 75, 50, 25? Would there ever be a reason to require more resolution

Reply:
from the datasheet theres not much more resolution